packages/consumption/src/modules/attributes/AttributesController.ts

```diff
@@ -47,6 +47,8 @@ import { IdentityAttributeQueryTranslator, RelationshipAttributeQueryTranslator,
 export class AttributesController extends ConsumptionBaseController {
     private attributes: SynchronizedCollection;
     private attributeTagClient: TagClient;
+    private cachedAttributeTagCollection?: AttributeTagCollection;
+    private cachedAttributeTagCollectionTimestamp?: CoreDate;
```

Comment:
that collection is lost when restarting the app, this is not good, b/c it looses the offline functionality.

Also the cache should be not lazily loaded, instead prepared for usage.